Neos.ContentGraph.DoctrineDbalAdapter/src/DoctrineDbalContentGraphProjection.php

```diff
@@ -807,7 +807,7 @@ function (NodeRecord $node) use ($eventEnvelope) {
                         'nodeanchorpoint' => $nodeAnchorPoint?->value,
                         'destinationnodeaggregateid' => $reference->targetNodeAggregateId->value,
                         'properties' => $reference->properties
-                            ? \json_encode($reference->properties, JSON_THROW_ON_ERROR)
```

Comment:
I can remove this one, overlooked it, the idea is to not create empty array in the DB `[]` but always objects `{}` seems cleaner, but it not necessary for aynthing I did below for now.